tests/resources/generator/algebraic_eqn_with_one_non_isolated_unknown/model.cellml

```diff
@@ -0,0 +1,48 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<model name="my_model" xmlns="http://www.cellml.org/cellml/2.0#" xmlns:cellml="http://www.cellml.org/cellml/2.0#">
+    <!-- Algebraic equation with one non-isolated unknown
+    Variables:
+     • a: 1 -> 9
+     • b
+     • c
+     • d
```

Comment:
The initial states of these variables are not given in this comment.

Reply:
Those comments list all the variables and, if they are an NLA-based variable, their initial guess and final value. In the model you mention, `a` is an NLA-based variable where the initial guess is `1` and the final value, after a call to KINSOL, is `9`. `b`, `c`, and `d` are just "normal" algebraic variables, hence no initial value or initial guess.

Reply:
FWIW, I originally did that so that I would have the relevant information at a glance rather than having to re-read the MathML code and remind myself of the mathematical model.